gen/src/DeposOrBust.cxx

```diff
@@ -63,9 +65,11 @@ bool Gen::DeposOrBust::operator()(input_queues& iqs, output_queues& oqs)
         }
 
         if (ds->depos()->empty()) {
+            log->debug("DeposOrBust: got empty deposet");
             get<1>(oqs).push_back(std::make_shared<EmptyFrame>(ds->ident()));
         }
         else {
+            log->debug("DeposOrBust: forward deposet");
```

Comment:
This log message would be more useful if it included some unique info.  Eg, the number of depos in the set.